src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -310,13 +310,37 @@ function getDrupalContent(buildOptions) {
     return () => {};
   }
 
+  const drupalDataWithBifurcatedLovellPages = drupalData => {
```

Comment:
I am a big fan of functions having verbs and properties being nouns.
Would something like this be more meaningful?
```suggestion
  const appendDrupalDataWithLovellTricarePages = drupalData => {
```